wscript

```diff
@@ -84,6 +81,7 @@ def configure(conf):
   conf.check_cxx(lib = 'msgpack')
   conf.check_cxx(lib = 'jubatus_mpio')
   conf.check_cxx(lib = 'jubatus_msgpack-rpc')
+  conf.check_cxx(lib = 'dl', mandatory = False)
```

Comment:
Note: `mandatory = False` is needed to keep FreeBSD compatibility. See also: #1099